src/odemis/driver/test/tescan_test.py

```diff
@@ -133,6 +145,177 @@ def test_pickle(self):
         sem.terminate()
         daemon.shutdown()
 
+
+class TestSEMRotateStage(unittest.TestCase):
+
+    @classmethod
+    def setUpClass(cls):
+        if TEST_NOHW:
+            return
+        cls.sem = tescan.SEM(**CONFIG_SEM_AMBER)
+
+        for child in cls.sem.children.value:
+            if child.name == CONFIG_SED["name"]:
+                cls.sed = child
+            elif child.name == CONFIG_SCANNER["name"]:
+                cls.scanner = child
+            elif child.name == CONFIG_STG["name"]:
+                cls.stage = child
+            elif child.name == CONFIG_FOCUS["name"]:
+                cls.focus = child
+
+    @classmethod
+    def tearDownClass(cls):
+        if TEST_NOHW:
+            return
+
+        cls.sem.terminate()
+        time.sleep(3)
+
+    def setUp(self):
+        if TEST_NOHW:
+            self.skipTest("No hardware present")
+
+    def tearDown(self):
+        pass
+
+    def test_move_abs(self):
+        """
+        Test if it's possible to move the stage in absolute coordinates
+        In addition also test the edge of the range of the rz axis
+        """
+        start_pos = self.stage.position.value.copy()
+        new_pos = {"x": start_pos["x"] + 5e-4, "y": start_pos["y"] + 10e-4, "z": start_pos["z"] + 12e-5,
+                   "rx": start_pos["rx"] - math.radians(2.5), "rz": start_pos["rz"] + math.radians(3.5)}
+
+        f = self.stage.moveAbs(new_pos)
+        f.result()
+        testing.assert_pos_almost_equal(self.stage.position.value, new_pos)
+
+        # move exactly on the edge of the max range of the rz axis
+        new_pos["rz"] = math.radians(360)
+        f = self.stage.moveAbs(new_pos)
+        f.result()
+        new_pos["rz"] = 0.0
+        testing.assert_pos_almost_equal(self.stage.position.value, new_pos)
```

Comment:
can you add a comment explaining why moving to 360 degrees, actually moves to zero degrees?